frontend/tests/jest/utils.tsx

```diff
@@ -157,3 +163,29 @@ export const getProductWithWarranty = (
 
    return product;
 };
+
+export const getReviewsResponse = (
+   type: 'filled' | 'empty' = 'filled'
+): ProductReviewData => {
+   let reviews: ProductReview[] = mockedReviews;
+
+   if (type === 'empty') {
+      reviews = [];
+   }
+
+   return {
+      reviews: reviews,
+      count: reviews.length,
+      average:
+         // @ts-ignore
```

Comment:
Why do we need the ts-ignore here?

Reply:
I was getting `possibly 'undefined'` error:
<img width="771" alt="Screen Shot 2022-12-20 at 12 10 35 PM" src="https://user-images.githubusercontent.com/58952979/208757389-46260b33-0d46-4576-a73c-344d5b8d7449.png">